src/ui/statusBarActionView.ts

```diff
@@ -0,0 +1,71 @@
+import { window, commands } from 'vscode';
+import { DafnyCommands } from '../commands';
+import * as vscode from 'vscode';
+
+export const fileIssueURL = 'https://github.com/dafny-lang/ide-vscode/issues/new?title='
++ encodeURIComponent('Verification issue')
++ '&body='
++ encodeURIComponent(`
+Failing code
+------------
+
+<!--paste relevant code below-->
+\`\`\`dafny
+
+\`\`\`
+
+Steps to reproduce the issue
+----------------------------
+
+* 
+
+Expected behavior
+-----------------
+
+
+Actual behavior
+---------------
+
+
+`.trimStart());
```

Comment:
We could create a template in the repo for this?

Reply:
Good point. To make it clear why it's better to generate it from VSCode, I also included the version of Dafny and the version of the VSCode extension in the last commit.